user-operator/src/main/java/io/strimzi/operator/user/model/KafkaUserModel.java

```diff
@@ -95,6 +97,11 @@ public static KafkaUserModel fromCrd(CertManager certManager,
         result.setAuthentication(kafkaUser.getSpec().getAuthentication());
 
         if (kafkaUser.getSpec().getAuthentication() instanceof KafkaUserTlsClientAuthentication) {
+            //if (kafkaUser.getMetadata().getName().length() > 64)    {
```

Comment:
Can I remove this line in some non-related PR?

Reply:
sure, why not.